charts/sn-platform/templates/bookkeeper/bookkeeper-cluster.yaml

```diff
@@ -66,7 +66,10 @@ spec:
       {{- if and .Values.affinity.anti_affinity .Values.bookkeeper.affinity.anti_affinity }}
       podAntiAffinity:
         {{ .Values.bookkeeper.affinity.type }}:
-          {{ if eq .Values.bookkeeper.affinity.type "requiredDuringSchedulingIgnoredDuringExecution" }}
+          {{- if eq .Values.bookkeeper.affinity.type "requiredDuringSchedulingIgnoredDuringExecution" }}
+          {{- with .Values.bookkeeper.affinity.podAffinityTerms }}
```

Comment:
Would it be better to move these custom affinity terms out of the `if` block? @ericsyh